packages/slate-editor/src/components/Toolbox/Toolbox.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import type { OptionsGroupOption } from '#components';
+import { VStack } from '#components';
+import { Toolbox, Toggle, OptionsGroup, Button, Link } from '#components';
+import { ExternalLink, Delete, ItemsLayoutVertical, ItemsLayoutHorizontal } from '#icons';
+
+export default {
+    title: 'Components/Toolbox',
+};
+
+export function Base() {
+    const [cardLayout, setCardLayout] = React.useState<
+        typeof cardLayoutOptions[number]['value'] | undefined
+    >('vertical');
+
+    const cardLayoutOptions: OptionsGroupOption<'vertical' | 'horizontal'>[] = [
+        {
+            value: 'vertical',
+            label: 'Vertical',
+            Icon: (props) => <ItemsLayoutVertical fill={props.isActive ? '#F9CA7B' : 'white'} />,
+        },
+        {
+            value: 'horizontal',
+            label: 'Horizontal',
+            Icon: (props) => <ItemsLayoutHorizontal fill={props.isActive ? '#F9CA7B' : 'white'} />,
+        },
+    ];
+
+    return (
+        <div style={{ width: 280 }}>
+            <Toolbox.Panel>
+                <Toolbox.Header withCloseButton>Web bookmark settings</Toolbox.Header>
+
+                <Toolbox.Section noPadding>
+                    <Link href="#" Icon={ExternalLink} fullWidth>
+                        View
+                    </Link>
+                </Toolbox.Section>
+
+                <Toolbox.Section caption="Preview image">
+                    <Toggle>Show preview image</Toggle>
+                </Toolbox.Section>
+
+                <Toolbox.Section caption="Card layout">
+                    <VStack spacing="spacing-1-5">
+                        <OptionsGroup
+                            name="card-layout"
+                            type="radio"
+                            options={cardLayoutOptions}
+                            selected={cardLayout}
+                            onChange={setCardLayout}
+                            columns={3}
+                        />
+                        <Toggle>Open in new tab</Toggle>
+                    </VStack>
+                </Toolbox.Section>
+
+                <Toolbox.Footer>
+                    <Button variant="clear" Icon={Delete} fullWidth>
+                        Remove web bookmark
+                    </Button>
+                </Toolbox.Footer>
+            </Toolbox.Panel>
```

Comment:
I did an extensive QA on the final look & feel here and noticed a few differences. Tried to outline them all here:
![image](https://user-images.githubusercontent.com/370680/153572642-1c9dc1e4-1d00-47dc-9877-759fd5bc35b6.png)

Smaller text (toggles labels, options labels, secondary buttons) all have font-size 13px in the designs.

Here's the design for the toggle "off" state: https://www.figma.com/file/KD30VWNjFh6mVWkg6Taqhe/06-%E2%80%93-Editor?node-id=1955%3A16012

Also see this: MT-4692